linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/JSGen.scala

```diff
@@ -89,10 +89,12 @@ private[emitter] final class JSGen(val config: Emitter.Config) {
     val descriptorTree =
         ObjectConstr(descriptor.map(x => StringLiteral(x._1) -> x._2))
 
-    val tree = Apply(
-        genIdentBracketSelect(VarRef(Ident("Object")), "defineProperty"),
-        List(obj, prop, descriptorTree))
-
-    WithGlobals(tree, Set("Object"))
+    globalRef("Object").map { objRef =>
```

Comment:
This is in the commit for "exports" but it's about "Object".

Reply:
It just uses the newly introduced helper as well. Do you want me to put it in a separate commit?

Reply:
Ah right, it was already tracked, but not using the helper. OK that's good as is, then.